packages/@divvi/mobile/src/analytics/Events.tsx

```diff
@@ -533,6 +533,8 @@ export enum SwapEvents {
   swap_show_fund_your_wallet = 'swap_show_fund_your_wallet',
   swap_add_funds = 'swap_add_funds',
   swap_switch_tokens = 'swap_switch_tokens',
+  swap_unfavorable_rate_warning_displayed = 'swap_unfavorable_rate_warning_displayed',
+  swap_unfavorable_rate_warning_cancelled = 'swap_unfavorable_rate_warning_cancelled',
```

Comment:
went with `warning` over `bottom_sheet` so this stays within the 40 char limit. Open to suggestions for other names